xls/modules/zstd/zstd_dec_test.cc

```diff
@@ -0,0 +1,267 @@
+// Copyright 2020 The XLS Authors
```

Comment:
can you update the copyright notice here and elsewhere (it should be `2024` for new files)

Reply:
All copyright notices should be good now